.python-version

```diff
@@ -0,0 +1 @@
+3.9.19
```

Comment:
what's this used for?

Reply:
This can be removed and the file added to the gitignore.